_drafts/mlbench-vs-mlperf.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: Comparison of MLBench and MLPerf
```

Comment:
If you could add yourself as author that would be nice. You need to add your entry in `data/authors.yml`